### PR TITLE
feat: implemented quilljs editor & showing formatted text on EventDetailPage

### PR DESCRIPTION
**Pull request by Markkos89**
## Description

- implemented quilljs 
- showing the quill formatted html on event details page

## Additional Information

- [ ] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #{issue number}_

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:

### CODE REVIEW

File: packages/nextjs/app/create-event/page.tsx [hunk 1]
```diff
@@ -183,6 +143,24 @@ const CreateQuizForm: React.FC = () => {
     }));
   };
 
+  const { quill, quillRef } = useQuill();
+
+  React.useEffect(() => {
+    if (quill) {
+      quill.on("text-change", () => {
+        // console.log("Text change!");
```
Comment (shyam-patel-kira) [inline packages/nextjs/app/create-event/page.tsx]: can you remove these comments

**Comment by HabacucMX** [comment]
@Markkos89 please add a redirect to home after creating an event.

**Comment by Markkos89** [comment]
> @Markkos89 please add a redirect to home after creating an event.

added :+1:

**Review by HabacucMX — APPROVED** [review]

File: packages/nextjs/app/create-event/page.tsx (hunk 1 above)
Comment (Markkos89) [reply packages/nextjs/app/create-event/page.tsx]: > can you remove these comments

done

**Pull request merged**